com.github.qarmin.Czkawka.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.qarmin.Czkawka
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: czkawka_gui
+rename-icon: com.github.qarmin.czkawka
+rename-desktop-file: com.github.qarmin.czkawka.desktop
```

Comment:
```suggestion
```
you already install them with the proper name?

Reply:
The name is `com.github.qarmin.Czkawka` (with the capital C), so I made it rename.

Reply:
You should definitely name them the same thing. So either rename the app-id or the files.

Reply:
> So either rename [...] the files

In this case, `rename-icon` and `rename-desktop-file` rename the files right?

Reply:
What I meant here is to rename them upstream to be consistent. Or rename the app-id & flatpak manifest to use a lower case as well.

Reply:
@qarmin what do you think?

Reply:
It's just a matter of naming, so for me it's completely indifferent whether Czkawka have first letter capitalized or lowercase.

Reply:
I guess you can rename everything to `com.github.qarmin.czkawka` since it'll just be easier to change.

Sorry for the bad naming, though.

Reply:
So after renaming it is safe to remove files about rename?

Reply:
Correct.